spec/models/accessible_format_request_model_spec.rb

```diff
@@ -0,0 +1,123 @@
+require "rails_helper"
+
+RSpec.describe AccessibleFormatRequest, type: :model do
+  subject(:accessible_format_request) { described_class.new(request_options) }
+  let(:document_title) { "An example title" }
+  let(:publication_path) { "/example/path" }
+  let(:format_type) { "large-print" }
+  let(:contact_name) { "J Doe" }
+  let(:contact_email) { "test@example.com" }
+  let(:alternative_format_email) { "department@example.com" }
+  let(:request_options) do
+    {
+      document_title: document_title,
+      publication_path: publication_path,
+      format_type: format_type,
+      contact_name: contact_name,
+      contact_email: contact_email,
+      alternative_format_email: alternative_format_email,
+    }
+  end
+
+  describe "validations" do
+    it { is_expected.not_to allow_value(nil).for(:document_title) }
```

Comment:
Half-agree? As we're passing this on to a system that's known to be finicky about inputs (as I understand it), it probably doesn't hurt to have these to confirm we've not knocked anything out by (eg) removing the email gem. If they were slow tests maybe another matter, but I don't think this adds much complexity and it might catch something.